src/ng/http.js

```diff
@@ -1268,9 +1268,9 @@ function $HttpProvider() {
           } else {
             // serving from cache
             if (isArray(cachedResp)) {
-              resolvePromise(cachedResp[1], cachedResp[0], shallowCopy(cachedResp[2]), cachedResp[3]);
+              resolvePromise(cachedResp[1], cachedResp[0], shallowCopy(cachedResp[2]), cachedResp[3], cachedResp[4]);
             } else {
-              resolvePromise(cachedResp, 200, {}, 'OK');
+              resolvePromise(cachedResp, 200, {}, 'OK', 'Request Completed');
```

Comment:
Since this strings just represent values (like an enum), not user messages, I think we should keep them as concise as possible. I propose:

- success
- error
- timeout
- abort